features/step_definitions/readDocument.js

```diff
@@ -102,8 +102,7 @@ var apiSteps = function () {
   });
 
   this.Then(/^I ?(don't)* find a document with "([^"]*)"(?: in field "([^"]*)")?(?: in index "([^"]*)")?$/, function (dont, value, field, index) {
-    var filters = {filter: { term: {}}};
-    filters.filter.term[field] = value;
+    var filters = {filter: { term: { [field]: value }}};
```

Comment:
This test read documents using Elasticsearch, while we replaced "term" with "equals" only for our real-time engine.